commands/config_e2e_test.go

```diff
@@ -111,3 +115,60 @@ func (s *MmctlE2ETestSuite) TestConfigSetCmd() {
 		s.Require().Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestConfigEditCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Edit a key in config", func(c client.Client) {
+		printer.Clean()
+
+		// check the value before editing
+		args := []string{"ServiceSettings.EnableSVGs"}
+		err := configGetCmdF(c, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().False(*printer.GetLines()[0].(*bool))
+		s.Require().Len(printer.GetErrorLines(), 0)
+		printer.Clean()
+
+		// create a shell script to edit config
+		content := `#! /bin/bash
+sed -i'old' 's/\"EnableSVGs\": false/\"EnableSVGs\": true/' $1
+rm $1'old'`
+
+		file, err := ioutil.TempFile(os.TempDir(), "config_edit_*.sh")
+		s.Require().Nil(err)
+		defer func() {
+			file.Close()
+			os.Remove(file.Name())
+			resetCmd := &cobra.Command{}
+			resetCmd.Flags().Bool("confirm", true, "")
+			s.Require().Nil(configResetCmdF(c, resetCmd, []string{"ServiceSettings"}))
+		}()
+		_, err = file.Write([]byte(content))
+		s.Require().Nil(err)
+		editorCmd := exec.Command("chmod", "+x", file.Name()) //nolint:gosec
+		err = editorCmd.Run()
+		s.Require().Nil(err)
+
+		os.Setenv("EDITOR", file.Name())
+
+		// check the value after editing
+		err = configEditCmdF(c, nil, nil)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 1)
+		config, ok := printer.GetLines()[0].(*model.Config)
+		s.Require().True(ok)
+		s.Require().True(*config.ServiceSettings.EnableSVGs)
+	})
+
+	s.Run("Edit config value without permissions", func() {
+		printer.Clean()
+
+		err := configEditCmdF(s.th.Client, nil, nil)
+		s.Require().NotNil(err)
```

Comment:
We should check if we received the expected error here. Also you may want to utilize `s.Require().Error()` or `s.Require().EqualError()` method.